src/topic_based_system.cpp

```diff
@@ -156,6 +160,15 @@ CallbackReturn TopicBasedSystem::on_init(const hardware_interface::HardwareInfo&
   {
     sum_wrapped_joint_states_ = true;
   }
+  if (get_hardware_parameter("use_initial_states_as_initial_commands", "false") == "true")
```

Comment:
Did you cherry-pick these changes from https://github.com/PickNikRobotics/topic_based_ros2_control/pull/24/files? Please use git cherry-pick to retain commit authorship attribution to the original author

Reply:
Thanks for the review.
I didn't think I would get a response, so I had to copy other pull requests on my own.
I have never used git cherry-pick, but I will give it a try.

Reply:
Sorry for the late review, I'm hoping to get it merged today :)

Reply:
I'll review it now, but I still don't understand the use case, can you give me an example where it's needed?

Reply:
I misunderstood the cherry-pick in git. Does that mean I should delete the relevant part and re-commit?

Reply:
I deleted the relevant part and re-committed.